cmd/executor/echo/main.go

```diff
@@ -71,3 +77,21 @@ func main() {
 		},
 	})
 }
+
+func jsonSchema() string {
+	return heredoc.Doc(
+		fmt.Sprintf(`{
+			"$schema": "http://json-schema.org/draft-04/schema#",
+			"title": "botkube/echo",
+			"description": "%s",
+			"pluginType": "executor",
```

Comment:
this is not a valid use of JSONSchema. The `title`, `description` properties are officially supported. 

For GraphQL service, the PluginType you can take from the index YAML.